### PR TITLE
Update gromacs workflow

### PR DESCRIPTION
**Pull request by simonbray**
After updating the GROMACS tools the workflow here also needs to be updated: https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/pull/35

### CODE REVIEW

File: training/computational-chemistry/gromacs/gromacs-test.yml [hunk 1]
```diff
@@ -9,4 +9,7 @@
     gro_output:
       asserts:
         has_text:
-          text: "LYSOZYME in water"
\ No newline at end of file
+          text: "LYSOZYME in water"
+    xtc_output:
+      file: traj.xtc
```
Comment (bgruening) [inline training/computational-chemistry/gromacs/gromacs-test.yml]: Can we download this fle from somewhere? Zenodo, cargo-port etc ...

Comment (simonbray) [reply training/computational-chemistry/gromacs/gromacs-test.yml]: It seems a bit crazy to upload a 13 MB file to Zenodo just to do a `sim_size` check. Would be nice if there was a Galaxy test option like `check_size=13000000` but I know this has been discussed before.

**Comment by bgruening** [comment]
Can you maybe create an issue for this idea. I don't think we have done that.

**Comment by bgruening** [comment]
Merci!

**Comment by simonbray** [comment]
Thanks for the merge.

I made an issue already: https://github.com/galaxyproject/galaxy/issues/8838

**Pull request merged**